third_party/xla/xla/tests/BUILD

```diff
@@ -1102,11 +1099,8 @@ xla_test(
     shard_count = 20,
     tags = [
         "optonly",
-<<<<<<< HEAD
         "no_rocm", #TODO(rocm): TEMP, sync 24-06-24
```

Comment:
Can we re-enable dot tests now? I see they were passing on CI on 30/07 https://github.com/ROCm/frameworks-internal/issues/8124#issue-2321996330